### PR TITLE
LPS-140660 Raylife D2C Mobile Design for Property Info Page

### PR DESCRIPTION
**Pull request by alvarolealdl**

### CODE REVIEW

**Comment by kevenleone** [comment]
@alvarolealdl we can't move forward with this commit with merge

Can you remove it?

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js [hunk 1]
```diff
@@ -43,95 +52,139 @@ export function FormProperty({form}) {
...
-			/>
+		  	<MobileContainer
+				 {...mobileContainerProps}
+				 mobileSubSection={getMobileSubSection(
+					 SUBSECTION_KEYS.HOW_MANY_STORIES_IS_THIS_BUILDING
+				 )}
+			>	
+				<NumberControlledInput
+					control={control}
+					label="How many stories is this building?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js:79]: Use a variable for this, please.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js [hunk 2]
```diff
@@ -43,95 +52,139 @@ export function FormProperty({form}) {
...
-			/>
+		    <MobileContainer
+				 {...mobileContainerProps}
+				 mobileSubSection={getMobileSubSection(
+					 SUBSECTION_KEYS.HOW_MANY_SQUARE_FEET_OF_THE_BUILDING
+				 )}
+			>	
+				<SquareFeatControlledInput
+					control={control}
+					label="How many square feet of the building does your business occupy?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js:99]: Use a variable for this, please.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js [hunk 3]
```diff
@@ -43,95 +52,139 @@ export function FormProperty({form}) {
...
-			/>
+		 	<MobileContainer
+				 {...mobileContainerProps}
+				 mobileSubSection={getMobileSubSection(
+					 SUBSECTION_KEYS.HOW_MANY_TOTAL_SQUARE_FEET_IS_THE_BUILDING
+				 )}
+			>
+				<SquareFeatControlledInput
+					control={control}
+					label="How many total square feet is the building?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js:128]: Use a variable for this, please

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js [hunk 4]
```diff
@@ -43,95 +52,139 @@ export function FormProperty({form}) {
...
-			/>
+		 	<MobileContainer
+				 {...mobileContainerProps}
+				 mobileSubSection={getMobileSubSection(
+					 SUBSECTION_KEYS.WHAT_YEAR_WAS_THE_BUILDING_CONSTRUCTED
+				 )}
+			>
+				<YearControlledInput
+					control={control}
+					label="What year was the building constructed?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js:144]: Use a variable for this, please.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js [hunk 5]
```diff
@@ -43,95 +52,139 @@ export function FormProperty({form}) {
...
-			/>
+		  	<MobileContainer
+				 {...mobileContainerProps}
+				 mobileSubSection={getMobileSubSection(
+					 SUBSECTION_KEYS.IS_THIS_THE_PRIMARY_LOCATION
+				 )}
+			>
+				<ControlledSwitch
+					control={control}
+					label="Is this the primary location where you conduct business?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Property.js:170]: Use a variable for this, please.

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :fire: But it needs some adjustments.

**Pull request closed without merging**